### ISSUE TITLE
TODO issues without a URL are appearing in Credo output twice

### ISSUE THREAD

**Issue by jcowgar** [theblitzapp/blitz_credo_checks#73]
Our configuration contains the TodoNeedsTicket check. I have double checked to ensure it does not exist in our .credo.exs file multiple times. The check works great, except that when a TODO item does not contain our ticket URL it appears in the credo output twice.

I do not see any information that is different between the two warnings. For this output below, I removed the URL from this TODO comment. When I put the URL back, both warnings disappear.

```
┃ [D] ↗ Found a TODO tag without a ticket URL in a comment: # TODO: Use a setup 
┃       domain query?

┃       Todos in codebase need an associated ticket URL

┃       i.e.

┃       

┃       # Todo: Make this better

┃       # https://linear.app/company/issue/443

┃       lib/project/file.ex:39 #(Project.File.has_ignore_rbr_option?)
┃ [D] ↗ Found a TODO tag without a ticket URL in a comment: # TODO: Use a setup 
┃       domain query?

┃       Todos in codebase need an associated ticket URL

┃       i.e.

┃       

┃       # Todo: Make this better

┃       # https://linear.app/company/issue/443

┃       lib/project/file.ex:39 #(Project.File.has_ignore_rbr_option?)
```

**Comment by alanvardy** [comment]
Thank you for the issue @jcowgar ! Looking into it now.

**Comment by alanvardy** [comment]
Fixed as part of v0.1.9 :+1:
